contrib/inventory/vmware_inventory.py

```diff
@@ -227,6 +228,10 @@ def read_settings(self):
                          'resourceconfig',
             'alias_pattern': '{{ config.name + "_" + config.uuid }}',
             'host_pattern': '{{ guest.ipaddress }}',
+            'use_hw_if_as_host': 'False',
+            # requires 'max_object_level': 4 (minimum) will check and set automatically if this is enabled.
+            'hw_if_ip_filter': '',
```

Comment:
You should be more explicit here. How does this filter work? Does it exclude or include the IP? Does it filter IP or IF name?